mppsolar/protocols/pi17.py

```diff
@@ -334,8 +337,8 @@
         "help": " -- queries enable/disable flag status from the Inverter",
         "type": "QUERY",
         "response": [
-            ["option", "Mute buzzer beep", ["Disabled", "Enabled"]],
-            ["option", "Mute buzzer beep in standby mode", ["Disabled", "Enabled"]],
+            ["option", "Mute buzzer beep", ["OFF", "ON"], {"subscribe":"PA", "ON":"PA1", "OFF":"PA0", "device-class": "outlet"}],
```

Comment:
I will post it in diffetent push request. The basic is, that HA will now that you can change the state of the variable and make it clickable. Then when you click it in HA, it will publish a message to specific topic with some payload. So this application will react to that (run a command to change some setting in the inverter).